os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/trace_uart.c

```diff
@@ -179,26 +169,27 @@ static uint32_t traceuart_dma_tx_complete(void *data)
     UART_RXDMACmd(TRACE_UART_DEV, DISABLE);
     /* Wake task to continue tx */
     t->tx_busy = 0;
-    if(t->tx_cb)
+    if (t->tx_cb)
         t->tx_cb();
     return 0;
 }
 #endif
+
 bool trace_uart_tx(uint8_t *pstr,uint16_t len, UART_TX_CB tx_cb)
 {
-    if(g_uart_obj.tx_switch == false)
+    if (g_uart_obj.tx_switch == false)
     {
-        if(tx_cb)
+        //dbg("trace_uart_deinit has been called please call init\r\n");
```

Comment:
Could you please unused log if it is not necessary?

Reply:
Done